docs/user/site_config.rst

```diff
@@ -0,0 +1,226 @@
+                       ['--ip-address', '10.10.10.55']]},
+
+The ``agent-class`` is given by the actual Agent which will be running. This
+must match the name defined in the Agent's code. The ``instance-id`` is a
+unique name given to this agent instance. Here we use the Lakeshore 372 serial
+number, `LSARR00`. Finally the arguments are used to pass default arguments to
+the Agent at startup, which contains the serial number again as well as the IP
+address of the 372.
+
+Environment Setup
+-----------------
+By default the system will look for site files in the path pointed to
+by environment variable ``OCS_CONFIG_DIR``. To define this, add the following
+to your ``.bashrc`` file::
+
+    export OCS_CONFIG_DIR='/path/to/ocs-site-configs/<your-institution-directory>/'
+
+The default site filename is ``default.yaml``.  In practice, it is recommended
+to name the configuration file after a given site, i.e. ``yale.yaml``, and symlink to
+``default.yaml``::
+
+    $ ln -s yale.yaml default.yaml
+
+During development, multiple YAML files may be in active use; then users will
+identify their config file through command line arguments when launching Agents
+and Control Clients (see below).
+
+.. note::
+    If you're proceeding in the same terminal don't forget to source your
+    ``.bashrc`` file.
+
+Commandline Arguments
+=====================
+There are several built in commandline arguments that can be passed to Agents
+when running. Agent Developers can also add custom arguments to their Agents.
+If running an Agent directly on a host these can be thrown when running the
+Agent manually, or configured in the 'arguments' section of your SCF. The built
+in arguments for all Agents are listed here, followed by some examples.
+
+.. note::
+    OCS users deploying Agents within Docker containers should be aware that
+    commandline options may be thrown by default within the Docker container. These
+    can be overridden by a user within their `docker-compose.yaml` file using
+    the CMD instruction.
+
+.. argparse::
+    :ref: ocs.site_config.add_arguments
+    :prog:
+
+.. _ocs_agent_cmdline_examples:
+
+Examples
+--------
+In the following examples, consider the "LS240_agent.py", which implements an
+Agent for talking to Lakeshore240 devices.  Suppose these are being run on a
+host called "host-1".  Refer to the example site configuration listed above.
+*(Note that to run these in the example tree you will usually need to add the
+options that select the example SCF and host, namely:* ``--site-file
+telescope.yaml --site-host host-1`` *. One exception to this is when using*
+``--site=none``. *)*
+
+1. Because there are two instances of "Lakeshore240Agent" registered
+   in the SCF, we must somehow pick one when running the agent::
+
+     $ python LS240_agent.py --instance-id=thermo1
+     I am in charge of device with serial number: LSA11AA
+
+2. We can ask our agent to connect to a different WAMP realm, for
```

Comment:
Out of curiosity is there any case which someone would actually do this?

Reply:
Probably not. This section was straight from the old docs on site configuration, I just made the examples a little more concrete (`LS240_agent.py` vs `river_agent.py`.)

Do you think it should be removed?

Reply:
I think it can be left in, I was just wondering.